CHANGELOG.md

```diff
@@ -1,3 +1,463 @@
+# 85.10.0
+
+This minor release expands support for trait collection and Material Elevation
+changes in several components. Bottom Sheet now supports shadows and elevation.
```

Comment:
If we are going to mention bottom sheet's additional support should we also add NavDrawer's support?

Reply:
Done.